mds/apis/prv_api/service_areas.py

```diff
@@ -29,17 +30,21 @@ class Meta:
         model = models.Polygon
 
     def create(self, validated_data):
+        geom = json.dumps(validated_data["geom"])
+        geo_object = geos.GEOSGeometry(geom)
```

Comment:
What's the type of `validated_data["geom"]`? GEOSGeometry can't handle it?

Reply:
it seems :slightly_smiling_face:

Reply:
or not :cry:

Reply:
Damn...

Reply:
So, it lacks some tests...